components/GlobalStyle/GlobalStyle.ts

```diff
@@ -76,6 +76,10 @@ html {
   }
 }
 
+html, body, #__next {
```

Comment:
I thought I had included this already but it seems not. This lets us do `height:100%` to take up the full height of the viewport.